### PR TITLE
rearrange requirements alphabetically

### PR DESCRIPTION
**Pull request by Adrian-Tamas**

### CODE REVIEW

**Comment by marco-c** [comment]
Looks good, as part of this could you also add `requirements-txt-fixer` as a pre-commit hook (under `repo: https://github.com/pre-commit/pre-commit-hooks`?

**Comment by Adrian-Tamas** [comment]
@marco-c sure. I will look into it but it will take a bit of time as I have not used the pre-commit-hooks before. So i'll be doing some reading up and start working on it.

**Comment by marco-c** [comment]
It's pretty simple, just need to add `requirements-txt-fixer` to the list here https://github.com/mozilla/bugbug/blob/323d2f583b3d666f3bb4573bc076b4f3197d8b6c/.pre-commit-config.yaml#L21.

File: infra/spawn_pipeline_requirements.txt [hunk 1]
```diff
@@ -1,4 +1,3 @@
-taskcluster==16.2.0
 json-e==3.0.0
-requests==2.22.0
-pyyaml==5.1.2
\ No newline at end of file
+pyyaml==5.1.2requests==2.22.0
```
Comment (marco-c) [inline infra/spawn_pipeline_requirements.txt]: There is an error here

Comment (Adrian-Tamas) [reply infra/spawn_pipeline_requirements.txt]: fixed. Sorry, that got modified by the hook because I missed the file and did not realize it was affected

**Pull request merged**